server/bleep/src/semantic.rs

```diff
@@ -65,29 +67,38 @@ macro_rules! val_parse_str(($hash:ident, $val:expr) => {
         .unwrap()
 });
 
-impl<'a> Payload<'a> {
-    pub fn from_qdrant(orig: ScoredPoint) -> Payload<'static> {
+impl Payload {
+    pub fn from_qdrant(orig: ScoredPoint) -> Payload {
         let ScoredPoint {
+            id,
             payload,
             score,
             vectors,
             ..
         } = orig;
 
-        let mut converted = payload
-            .into_iter()
-            .map(|(key, value)| (key, kind_to_value(value.kind)))
-            .collect::<HashMap<String, serde_json::Value>>();
+        let Some(PointId { point_id_options: Some(PointIdOptions::Uuid(id)) }) = id
+	else {
+	    // unless the db was corrupted/written by someone else,
```

Comment:
By `db` are we referring to `Qdrant` here?

Reply:
Yes.